xl_auth/permission/forms.py

```diff
@@ -44,6 +45,9 @@ def __init__(self, current_user, *args, **kwargs):
                 key=lambda _: _[1]
             )
 
+        if not current_user.is_admin:
```

Comment:
I would prefer having this as part of the validation step instead, returning a `ValidationError` if someone without correct permission tries to change the value. This would require including a hidden field in the forms, but I think that's fine.

The reason why I prefer the other way is that it's more explicit since we treat the submission as invalid, rather than silently dropping fields.

Reply:
Ok. See commit.